### DISCUSSION TITLE
I cannot set my old username

### DISCUSSION THREAD

**Discussion by dizdep** [community/community#119610] [Other Feature Feedback, Questions, & Ideas] [answered]
I deleted second profile with username ibobro and after that i wanted to set that username to my first profile, but it was unavailable. What i should do? Or what i did wrong?

**Comment by ettaboyle** [comment, marked as answer]
Hi @yourvows! The best (and only) place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). We cannot respond to any account specific questions in this public forum. 

Please open a ticket with them :)
